app/views/ShareView/Header.js

```diff
@@ -64,6 +65,13 @@ const Header = React.memo(({ room, thread, theme }) => {
 
 	const textColor = themes[theme].previewTintColor;
 
+	let title;
+	if (thread?.id) {
+		title = makeThreadName(thread);
+	} else {
+		title = RocketChat.getRoomTitle(room);
+	}
```

Comment:
It's not needed, since it'll never be a thread room..

Reply:
Of course it will.
When you share a file inside a thread, it's going to open this screen.